chains/manager.go

```diff
@@ -266,12 +271,17 @@ type manager struct {
 // New returns a new Manager
 func New(config *ManagerConfig) Manager {
 	return &manager{
-		Aliaser:                ids.NewAliaser(),
-		ManagerConfig:          *config,
-		stakingSigner:          config.StakingTLSCert.PrivateKey.(crypto.Signer),
-		stakingCert:            staking.CertificateFromX509(config.StakingTLSCert.Leaf),
-		subnets:                make(map[ids.ID]subnets.Subnet),
-		chains:                 make(map[ids.ID]handler.Handler),
+		Aliaser:       ids.NewAliaser(),
+		ManagerConfig: *config,
+		stakingSigner: config.StakingTLSCert.PrivateKey.(crypto.Signer),
+		stakingCert:   staking.CertificateFromX509(config.StakingTLSCert.Leaf),
+		subnets:       make(map[ids.ID]subnets.Subnet),
+		chains:        make(map[ids.ID]handler.Handler),
+		fxs: map[ids.ID]fx.Factory{
+			secp256k1fx.ID: &secp256k1fx.Factory{},
+			nftfx.ID:       &nftfx.Factory{},
+			propertyfx.ID:  &propertyfx.Factory{},
+		},
```

Comment:
Should this just be a global? It seems weird to have this be here even though its a constant.